lua/neotest-python/base.lua

```diff
@@ -72,6 +72,13 @@ function M.get_python_command(root)
     end
   end
 
+  if lib.files.exists("uv.lock") then
+    -- I'm not sure if there's a way to get non-default venv
```

Comment:
You could run `uv run python -c 'import sys; print(sys.executable)'` here to get the python path

Reply:
Thanks! That's a neat trick, seems to be working.

Reply:
@rcarriga done 🙌